### DISCUSSION TITLE
The documentation link is not working.

### DISCUSSION THREAD

**Discussion by NiUr27** [Virtual-FCS/VirtualFCS#32] [General]
"Documentation for the VirtualFCS library is maintained regularly and available online at the URL: https://virtual-fcs.github.io/VirtualFCS/ "

You get the error message:
 404
There isn't a GitHub Pages site here.

**Comment by mikegerhardt** [comment]
Hi, thanks for letting us know. Documentation is available in the Documentation Browser in the VirtualFCS package when you open it in the OM Connection Editor. I'll take down the broken link for now.
